docs/getting-started-guides/mesos.md

```diff
@@ -42,9 +45,9 @@ The internal IP address of the master may be obtained via `hostname -i`.
 
 ```bash
 $ export servicehost=$(hostname -i)
-$ export mesos_master=${servicehost}:5050
 $ export KUBERNETES_MASTER=http://${servicehost}:8888
 ```
+
```

Comment:
The docker hub version is not maintained. With the follow-up commit we switch to the quai.io container with the same version as in cluster/images/etcd/Makefile (v2.0.9).